exercises/variable-length-quantity/cases_test.go

```diff
@@ -107,46 +107,55 @@ var decodeTestCases = []struct {
 	description string
 	input       []byte
 	output      []uint32 // nil slice indicates error expected.
+	shouldfail  bool
```

Comment:
To match the Go idiom, this should be mixed case, e.g. `shouldFail`. However, it would probably be even more idiomatic to have it be named `ok`, which would make the conditional around the assertion:

```go
if !tc.ok {
    // ...
}
```